modin/pandas/series.py

```diff
@@ -1258,8 +1258,10 @@ def mask(
         errors: IgnoreRaise | NoDefault = no_default,
         try_cast=no_default,
     ):  # noqa: PR01, RT01, D200
-        # This method exists purely because `errors` has different default value for Series :(
-        return super().mask(cond, other, inplace, axis, level, errors, try_cast)
+        # This method exists because `errors` has different default value for Series :(
+        return super(Series, self).mask(
+            cond, other, inplace, axis, level, "raise", try_cast
```

Comment:
Shouldn't we pass in `errors` here?

Reply:
Also seems this argument has been deprecated since 1.5? So we should be coercing types anyways?

Reply:
That deprecation is the reason why I'm not passing the `errors` through, as it is expected to always behave as if `raise` is given.